learning/tour-of-beam/terraform-v2/environments/uat/main.tf

```diff
@@ -0,0 +1,37 @@
+locals {
+  env = "uat"
+}
+
+provider "google" {
+  project = var.project_id
+}
+
+module "iam" {
+  source = "../../modules/iam"
+  project_id = var.project_id
+  service_account_id = var.service_account_id
+  depends_on = [module.api_enable]
+}
+
+module "buckets" {
+  source = "../../modules/buckets"
+  project_id = var.project_id
+  bucket_name = var.bucket_name
+  depends_on = [module.iam, module.api_enable]
+}
+
+module "api_enable" {
+  source = "../../modules/api_enable"
+  project_id = var.project_id
+}
+
+module "cloud_functions" {
```

Comment:
why do we keep it in `environments`?
I thought `environments` is where only runtime variables defined

Reply:
That's correct, my mistake. 
Action: To move files out of "environments"